readme-update (#94)
PiotrRynio/CodersCamp2021.Project.JavaScript: README.md

# Breaking Bad Quiz

**Demo on Netlify:** [LINK](https://breaking-bad-quiz-game.netlify.app/)

## Table of contents

- [General Info](#general-information)
- [Our Crew](#our-crew)
- [Technologies Used](#technologies-used)
- [Screenshots](#screenshots)
- [Running the project](#running-the-project)
- [Project Status](#project-status)

## General Information

This is a quiz type game made using the popular Breaking Bad series universe API with multiple levels of difficulties, ranking scoreboard and randomized questions created with HTML, SCSS and JavaScript. Project build for Coders Camp 2021/2022.

> This app has been created by a group of 6 people for a free coding program called CodersCamp.

Reviewer: ```suggestion
This app has been created by a group of 6 people during CodersCamp course.
```

Our motivation was to learn how to work as a group on a project and improve our skills in HTML, SCSS and JavaScript.
The purpose of this application is to learn the possibilites and boundaries of the mentioned above languages.

- API URL: [An API of XXX](XXX)
- Demo on Netlify: [LINK](https://breaking-bad-quiz-game.netlify.app)

> - Repo url: [LINK](XXX)

Reviewer: Tutaj

## Our Crew

Mentor: [Piotr Rynio](https://github.com/PiotrRynio)

- [Agnieszka Przybyłowska](https://github.com/APrzybylowska)
- [Patryk Święcicki](https://github.com/PSwiecicki)
- [Radosław Dulisz](https://github.com/RadekDulisz)
- [Bartosz Dombrow](https://github.com/BartoszDombrow)
- [Kuba Jarosz](https://github.com/kubajaro)
- [Jarosław Kaczor](https://github.com/jaroslawkaczor)

## Main functionalities:

1. Tournament organisation
2. XXX
3. ...XXX

> 

Reviewer: Uzupełnij tutaj
